src/core/utils.js

```diff
@@ -97,6 +97,10 @@ const fetchDestinations = new Set([
   "",
 ]);
 
+//these classes mean a section is non-normative
```

Comment:
As "section" is an element, maybe something like: 
```suggestion
 // CSS classes that indicate that a container element is non-normative.
```

Reply:
"a container element" somehow sounds like "a parent element" though.

Reply:
True, we can probably drop "container".

Reply:
Updated suggestion.

Reply:
How about:

```diff
- that indicate that
+ indicating
```

as IMO doubled "that" looks weird 😀